src/Frontend/Modules/Location/Layout/Templates/Index.html.twig

```diff
@@ -25,9 +25,7 @@
 
             <div itemprop="name" class="hidden"><strong>{{ locationItem.title }}</strong></div>
             <address itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
-              <span itemprop="streetAddress">{{ locationItem.street }} {{ locationItem.number }}</span><br>
-              <span itemprop="postalCode">{{ locationItem.zip }}</span>
-              <span itemprop="addressLocality">{{ locationItem.city }}</span><br>
+              <span itemprop="streetAddress">{{ locationItem.address }}</span><br>
```

Comment:
if that is the case then it looks to me that this is a bad solution to the problem at hand